packages/ruby/index.ts

```diff
@@ -142,14 +139,7 @@ export async function build({
       } else {
         // try installing. this won't work if native extesions are required.
         // if that's the case, gems should be vendored locally before deploying.
-        try {
-          await bundleInstall(bundlerPath, bundleDir, gemfilePath);
-        } catch (err) {
-          debug(
-            'unable to build gems from Gemfile. vendor the gems locally with "bundle install --deployment" and retry.'
-          );
-          throw err;
-        }
+        await bundleInstall(bundlerPath, bundleDir, gemfilePath);
```

Comment:
That debug message is no longer relevant if an error occurs? Are there enough clues in the stdout + stderr to indicate how to proceed when an error occurs?

Reply:
It should have more information than before because we were previously hiding the logs of bundle install

Also see [df4f8ee](https://github.com/vercel/vercel/pull/8388/commits/df4f8eec75b7c93f66d405119a31f22ab4ded854)